Makefile

```diff
@@ -58,12 +58,20 @@ docker-push:
 kind-load:
 	kind load docker-image $(IMG) || { echo >&2 "kind not installed or error loading image: $(IMG)"; exit 1; }
 
-e2e-setup:
+e2e-setup-master:
```

Comment:
where is this used?

Reply:
It can be used later on. It is added on purpose.